apps/frontend/src/pages/access-management/styles.module.css

```diff
@@ -0,0 +1,22 @@
+.content {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	padding-left: 10%;
+}
+
+.title {
+	font-size: 30px;
+	font-weight: 600;
+	line-height: 120%;
+}
+
+.sub-title {
+	font-size: 24px;
+	font-weight: 600;
+	line-height: 120%;
+}
+
+.table-container {
+	width: 70%;
```

Comment:
Where this number comes from? You'd better remove this at all, it should have fill all available space (so `width: 100%`, which is defaults). Instead of this `.content` will have appropriate `padding`